packages/components/src/components/Callout/Callout.tsx

```diff
@@ -0,0 +1,137 @@
+import { SystemProp, Theme } from "@xstyled/styled-components";
+import React from "react";
+import { Box } from "../../primitives/Box";
+import { Icon, IconProps } from "../Icon";
+
+type CalloutVariantOptions = "error" | "information" | "success" | "warning";
+
+type CalloutProps = {
+  /** The contents of the callout. Can be text or valid text related HTML, i.e. anchor and strong elements. */
+  children: NonNullable<React.ReactNode>;
+
+  /** The type of callout. */
+  variant?: CalloutVariantOptions;
+};
+
+const getVariantProps = (
+  variant: CalloutVariantOptions
+): {
+  backgroundColor: SystemProp<keyof Theme["colors"], Theme>;
+} => {
+  switch (variant) {
+    case "error": {
+      return {
+        backgroundColor: "colorBackgroundError",
+      };
+    }
+    case "warning": {
+      return {
+        backgroundColor: "colorBackgroundWarning",
+      };
+    }
+    case "success": {
+      return {
+        backgroundColor: "colorBackgroundSuccess",
+      };
+    }
+    default: {
+      return {
+        backgroundColor: "colorBackgroundInfo",
+      };
+    }
+  }
+};
+
+const getIconVariantProps = (
```

Comment:
Why not create just one object with Icons and Variant props? Something like this:

```
{ 
   'error':  {
      'icon':  {
        color: "colorIconError",
        icon: "XCircleIcon",
      },
      'variant':  { backgroundColor: 'colorBackgroundError' }
   } 
}